src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -636,6 +646,101 @@ data class AssignedPlotPayload(
   )
 }
 
+data class BiomassAdditionalSpeciesPayload(
+    val isInvasive: Boolean,
+    val isThreatened: Boolean,
+    val speciesId: SpeciesId?,
+    val speciesName: String?,
+) {
+  fun toModel(): BiomassAdditionalSpeciesModel {
+    return BiomassAdditionalSpeciesModel(isInvasive, isThreatened, speciesId, speciesName)
+  }
+}
+
+data class BiomassMeasurementPayload(
+    val additionalSpecies: List<BiomassAdditionalSpeciesPayload>,
+    val description: String?,
+    val forestType: BiomassForestType,
+    @Schema(minimum = "0", maximum = "100") val herbaceousCoverPercent: BigDecimal,
```

Comment:
Could put a trailing `//` after the annotation to make all the `val` keywords line up. Same with the other annotations in this file that aren't long enough to make ktfmt put the `val` on a new line.